tools/mountsnoop.py

```diff
@@ -379,9 +408,9 @@ def print_event(mounts, umounts, cpu, data, size):
                     target=decode_mount_string(syscall['target']),
                     flags=decode_umount_flags(syscall['flags']),
                     retval=decode_errno(event.union.retval))
-            print('{:16} {:<7} {:<7} {:<11} {}'.format(
-                syscall['comm'].decode('utf-8', 'replace'), syscall['tgid'],
-                syscall['pid'], syscall['mnt_ns'], call))
+            print('{:16} {:<7} {:<7} {:<7} {:<11} {}'.format(
```

Comment:
Looks like this is the first time field ordering is being changed since `mountsnoop`'s [initial commit](https://github.com/iovisor/bcc/commit/e822a8189d9ba376112960417223191296417431). Is there any chance someone might be relying on the current order? I'm imagining a situation where someone is parsing the output of `mountsnoop` and doing `pid = fields[1]` instead of using first line to determine order.

I don't know if this is a reasonable concern for this tool even if it could happen. My hunch would be not to worry about it, but wanted to get your thoughts.

Reply:
The manpage says

> .SH STABILITY
> Unstable - in development.

as do all the other tool manpages

Reply:
Thank you very much.
Compatibility is very important. We really need to consider it carefully here. If we add the two fields later, will it be better? It's like this:
COMM PID TID MNT_ NS CALL PCOMM PPID

Reply:
This is a good idea. We'll optimize it later. 
Thanks.